doc/user-guide/hierarchical-data.rst

```diff
@@ -11,7 +9,6 @@ Working With Hierarchical Data
     import numpy as np
     import pandas as pd
     import xarray as xr
-    from datatree import DataTree
 
     np.random.seed(123456)
     np.set_printoptions(threshold=10)
```

Comment:
I can't leave a diff in the pre-existing lines below for some reason, but "be be" should replaced with "be" on this line:
```
and it can often be be useful to think of these in terms of a hierarchy of related groups of data.
```